Objects/stringlib/unicode_format.h

```diff
@@ -462,8 +462,17 @@ get_field_object(SubString *input, PyObject *args, PyObject *kwargs,
             tmp = getattr(obj, &name);
         else
             /* getitem lookup "[]" */
-            if (index == -1)
+            if (index == -1) {
```

Comment:
If negative indices are not allowed, the condition should be `index < 0`.

Reply:
`index` has a different meaning here. It is returned by `get_integer` and is either `Py_ssize_t` or `-1` if failed to parse substring as `Py_ssize_t`.

Reply:
> `index` has a different meaning here. It is returned by `get_integer` and is either `Py_ssize_t` or `-1` if failed to parse substring as `Py_ssize_t`.

Okay, I understand.